draft-foudil-securitytxt.md

```diff
@@ -550,42 +559,46 @@ organization operating that website to be a way to signal to researchers
 researchers reporting security issues to that organization.
 
-Therefore, implementors MUST NOT assume that presence or absence of
+Therefore, implementors shouldn't assume that presence or absence of
 a "security.txt" file grants or denies permission for security testing.
-Any such permission MAY be defined in a security or disclosure policy
-(as per {{policy}}) or a new directive (as per {{extensibility}}).
+Any such permission may be indicates in the company's vulnerability disclosure policy
+(as per {{policy}}) or a new field (as per {{extensibility}}).
 
 ## Multi-user Environments
 
 In multi-user / multi-tenant environments, it may possible for a user to take
-over the location of the "security.txt" file. Organizations SHOULD reserve
+over the location of the "security.txt" file. Organizations should reserve
 the "security.txt" namespace at the root to ensure no third-party can create a page with
 the "security.txt" AND "/.well-known/security.txt" names.
 
 ## Protecting Data in Transit
 
-To protect a "security.txt" file from being tampered with in transit, implementors MUST use
+To protect a "security.txt" file from being tampered with in transit, implementors should use
 HTTPS (as per {{!RFC2818}}) when serving the file itself and for retrieval of any web URLs
 referenced in it (except when otherwise noted in this specification). As part of the TLS
-handshake, implementors MUST validate the provided X.509 certificate
+handshake, implementors should validate the provided X.509 certificate
 in accordance with {{!RFC6125}} and the following considerations:
 
 - Matching is performed only against the DNS-ID identifiers.
 
 - DNS domain names in server certificates MAY contain the wildcard
 character '*' as the complete left-most label within the identifier.
 
-The certificate MAY be checked for revocation via the Online Certificate Status
+The certificate may also be checked for revocation via the Online Certificate Status
 Protocol (OCSP) {{!RFC6960}}, certificate revocation lists (CRLs), or similar mechanisms.
 
-As an additional layer of protection, it is also RECOMMENDED that
+In cases where the "security.txt" file cannot be served via HTTPS or is
+being served with an invalid certificate, additional human triage is recommended since
+the contents may have been modified while in transit.
+
+As an additional layer of protection, it is also recommended that
 organizations digitally sign their "security.txt" file with OpenPGP (as per {{signature}}).
 Also, to protect security reports from being tampered with or observed while in transit,
-organizations SHOULD specify encryption keys (as per {{encryption}}) unless
+organizations should specify encryption keys (as per {{encryption}}) unless
 HTTPS is being used.
 
 However, the determination of validity of such keys is out of scope
-for this specification. Implementors MUST establish other secure means to
+for this specification. Researches need to establish other secure means to
```

Comment:
```suggestion
for this specification. Security researches need to establish other secure means to
```

Reply:
will change

Reply:
Actually, there is a typo there. It should be:

```suggestion
for this specification. Security researchers need to establish other secure means to
```

Reply:
I fixed it